test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -210,9 +212,11 @@ func runCpaSampleCode(ctx context.Context, f *framework.Framework, runTests int,
 	framework.ExpectNoError(err, "pod Create API error")
 
 	ginkgo.By("waiting the cpa_sample_code pod for the resource" + resourceName.String() + "to finish successfully")
-	e2epod.NewPodClient(f).WaitForSuccess(ctx, pod.ObjectMeta.Name, 300*time.Second)
 
-	output, _ := e2epod.GetPodLogs(ctx, f.ClientSet, f.Namespace.Name, pod.ObjectMeta.Name, pod.Spec.Containers[0].Name)
-
-	framework.Logf("cpa_sample_code output:\n %s", output)
+	err = e2epod.WaitForPodSuccessInNamespaceTimeout(ctx, f.ClientSet, pod.ObjectMeta.Name, f.Namespace.Name, 300*time.Second)
+	if err != nil {
+		output, _ := e2epod.GetPodLogs(ctx, f.ClientSet, f.Namespace.Name, pod.ObjectMeta.Name, pod.Spec.Containers[0].Name)
+		framework.Logf("cpa_sample_code output:\n %s", output)
+		framework.Failf("unable to finish demo pod: %v", err)
+	}
```

Comment:
is this missing `gomega.Expect(err).To(gomega.BeNil(), utils.GetPodLogs(...))`

Reply:
Ah, yes, thanks.!